docs/getting-started/styling/StyledSystem.stories.mdx

```diff
@@ -29,7 +29,7 @@ To help achieve a consistent UI, Orbit [style props](#style-props) are based on
 
 ## Style props
 
-An Orbit style property is a mapping of a [CSS property](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Properties_Reference) to a component property. Orbit styled system let you set style values for CSS properties like [font-size](https://developer.mozilla.org/en-US/docs/Web/CSS/font-size), [margin](https://developer.mozilla.org/en-US/docs/Web/CSS/margin), [padding](https://developer.mozilla.org/en-US/docs/Web/CSS/padding), [width](https://developer.mozilla.org/en-US/docs/Web/CSS/width), and many more which we deem appropriate for ShareGate applications. Shorthands like `border` and `paddingX` are also provided to helps you save time. An exhaustive list of all the supported props is available in the [props reference](?path=/story/props-reference--page) page.
+An Orbit style property is a mapping of a [CSS property](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Properties_Reference) to a component property. Orbit styled system let you set style values for CSS properties like [font-size](https://developer.mozilla.org/en-US/docs/Web/CSS/font-size), [margin](https://developer.mozilla.org/en-US/docs/Web/CSS/margin), [padding](https://developer.mozilla.org/en-US/docs/Web/CSS/padding), [width](https://developer.mozilla.org/en-US/docs/Web/CSS/width), and many more which we deem appropriate for ShareGate applications. Shorthands like `border` and `paddingX` are also provided to helps you save typing. An exhaustive list of all the supported props is available in the [props reference](?path=/story/props-reference--page) page.
```

Comment:
My precious comment should have been on this line

Reply:
maybe my formulation isn’t right. “Saving time” didn’t made much sense because well you’ll save what, 45ms? It’s more about saving letters which you won’t have to type.

Reply:
Maybe “keystrokes” ?

Reply:
keystrokes is perfect